ci-operator/config/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10.yaml

```diff
@@ -31,6 +31,10 @@ tests:
   commands: make test
   container:
     from: test-bin
+- as: e2e-test
```

Comment:
Not required. The e2e tests are not available in the release-4.10 branch.

Reply:
Updated